.s2i/bin/assemble

```diff
@@ -0,0 +1,266 @@
+#!/bin/bash
```

Comment:
I want to understand this file more.  I know it's OpenShift's way of creating source-to-image containers.  Why is it in the root?  Shouldn't it exist in the `/api` since it's only related to that?  Why do we need this for this feature of db migration?

Reply:
Yeah, this is the unfortunate result of MS not creating their images with Openshift in mind. Without this the random uid assigned by openshift doesn't have permissions to execute the dotnet-ef tool installed during the build. See the comment slightly below this for more details. 

I think this could maybe go into /api? I hate how openshift automagically finds and detects these in source, it feels very unsafe to me.